pages/services/data-science-engine/1.0.0/hdfs/index.md

```diff
@@ -132,3 +132,52 @@ You can also specify them through the UI.
                   }
     }
     ```
+   
+# TensorBoard
```

Comment:
Should we put Tensorboard in a new page for `Tensorflow`? It doesn't feel like it should fall under HDFS. 

That way we can include information on Tensorflow and Tensorboard
@akirillov WDYT?